dizoo/atari/config/serial/pong/pong_trex_offppo_config.py

```diff
@@ -7,13 +7,15 @@
         evaluator_env_num=4,
         n_evaluator_episode=8,
         stop_value=20,
-        env_id='PongNoFrameskip-v4',
+        env_id='Pong-v4',
+        #'ALE/Pong-v5' is available. But special setting is needed after gym make.
         frame_stack=4,
     ),
     reward_model=dict(
         type='trex',
         algo_for_model='ppo',
-        env_id='PongNoFrameskip-v4',
+        env_id='Pong-v4',
```

Comment:
why trex reward_model need `env_id` here, maybe it is a redundant field, plz validate it, you can ask for @kxzxvbk 
 or @Will-Nie for help